mlrun/serving/routers.py

```diff
@@ -371,26 +467,32 @@ def _parallel_run(self, event: dict):
         return results
 
     @staticmethod
-    def init_pool(server_spec, routes):
+    def init_pool(server_spec, routes, object_id):
```

Comment:
`object_id` is the result of `id(route)`? If so, why? `results` in itself is a dictionary so the routes in it should be unique already, why the use of `id`?

Reply:
See answer above